deltachat-ios/Chat/Views/Cells/BaseMessageCell.swift

```diff
@@ -417,11 +417,19 @@ public class BaseMessageCell: UITableViewCell {
 
     private func attachPadlock(to text: NSMutableAttributedString, color: UIColor? = nil) {
         let imageAttachment = NSTextAttachment()
+        let cachedImageKey = "ic_lock\(color != nil ? "_color" : "")"
+        let cachedImage = ThumbnailCache.shared.restoreImage(key: cachedImageKey)
```

Comment:
ThumbnailCache is also used for gallery items and video thumbnails - they use a url that is guaranteed not to be never ic_lock or so?